docs/3.1. Discovery - Registered Operation.md

```diff
@@ -10,20 +10,20 @@ This document describes usage of NMOS APIs for discovery in cases where where a
 2. Node scans for an active Registration API on the network using unicast and/or multicast DNS service discovery (type '\_nmos-register.\_tcp')
    *  Nodes which support v1.2 and earlier versions MUST additionally browse for the deprecated service type '\_nmos-registration.\_tcp'.
 3. Given multiple returned Registration APIs, the Node orders these based on their advertised priority (TXT pri), filtering out any APIs which do not support its required API version and protocol (TXT api_ver and api_proto).
+   *  Where a Node supports multiple API versions simultaneously, see the [Upgrade Path](6.0.%20Upgrade%20Path.md) for additional requirements in filtering the discovered API list.
```

Comment:
@andrewbonney I think the next bullet (4) could be usefully clarified as well, e.g. inserting "of the same API version" after "multiple Registration APIs". I'm also not sure it's been helpful mentioning RFC 2782 here since af9b50d56974a442c3ba4a95511ff3529e252752 (v1.1).

(The same would then apply to the **Querying** section below.)